### PR TITLE
Fix alert rules of ticdc

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix the alert rule of `tikv_cdc_min_resolved_ts_no_change_for_1m` and add ticdc_changefeed_failed alert rule.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-alert-rules.md [hunk 1]
```diff
@@ -39,6 +39,20 @@ summary: 了解 TiCDC 集群监控报警规则以及处理方法。
...
 
+### `ticdc_changefeed_failed`
+
+* 报警规则：
+
+    `(max_over_time(ticdc_owner_status[1m]) == 2) > 0`
+
+* 规则描述：
+
+    TiCDC 某个同步任务遇到无法自动恢复的错误，进入 Failed 状态。
```
Comment (Oreoxmt) [inline ticdc/ticdc-alert-rules.md:50]:
```suggestion
    TiCDC 某个同步任务遇到无法自动恢复的错误，进入 failed 状态。
```
Keep it consistent with: https://github.com/pingcap/docs-cn/blob/a56b50776d9c1f416ec7b9a5b0755e53b284d1ae/ticdc/ticdc-open-api.md?plain=1#L253-L257

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@asddongmen @flowbehappy Could you please have a tech review? Thanks.

**Review by asddongmen — APPROVED** [review]

**Pull request merged**